internal/cli/atlas/serverless/update.go

```diff
@@ -71,8 +71,13 @@ func (opts *UpdateOpts) newServerlessUpdateRequestParams() *atlasv2.ServerlessIn
 		}
 	}
 
+	if len(opts.tag) > 0 {
+		params.Tags = []atlasv2.ResourceTag{}
+	}
 	for k, v := range opts.tag {
-		params.Tags = append(params.Tags, atlasv2.ResourceTag{Key: pointer.Get(k), Value: pointer.Get(v)})
+		if k != "" && v != "" {
+			params.Tags = append(params.Tags, atlasv2.ResourceTag{Key: pointer.Get(k), Value: pointer.Get(v)})
+		}
```

Comment:
Does this mean that tags are always appended now (no replace all on the update command?)

If yes, we should also update the docs - I think it's a more predictible behaviour

Reply:
it is a replace @colm-quinn we don't fetch current state